usagov/vote-gov-drupal
# Sprint 60: Register to Vote page updates

## Jira tickets

[VOTE-1938](https://cm-jira.usa.gov/browse/VOTE-1938)
[VOTE-1983](https://cm-jira.usa.gov/browse/VOTE-1983)
[VOTE-1109](https://cm-jira.usa.gov/browse/VOTE-1109)

## Description

1. Update the register to vote page based on the new designs (excluding the side navigation)
2. Fix Navajo font in register tool
3. Add aria labels to state selector

## Deployment and testing

### Post-deploy steps

1. lando retune
2. cd into theme, npm run build

### QA/Testing instructions

1. Sign in on your local, go to content -> Register to vote -> translate
2. Add a Navajo and Arabic page (you can keep this in English for testing)
3. Go to http://vote-gov.lndo.site/register and confirm the design matches the figma designs
4. Switch to Arabic and confirm right-left styling is correct
5. Switch to Navajo, confirm the heading in the register tool uses the Navajo font

## Checklist for the Developer




## Checklist for the Peer Reviewers
